commands/reload.js

```diff
@@ -12,19 +12,31 @@ class Reload extends Command {
   }
 
   async run(message, args, level) { // eslint-disable-line no-unused-vars
+    const module = args.join(" ");    
```

Comment:
Not sure why you would need to combine the args, as event and command names are, and should be only one word.

Reply:
Honestly, I'm not sure why I do this, will remove and adjust accordingly.

Reply:
That's fine, I just saw it as a needless change as all events/commands/slashies are one word.